TU-Agent/tygron.mod2g

```diff
@@ -3,13 +3,17 @@ use tygron as actionspec.
 use demolish as module.
 use buildConstruction as module.
 use buyLand as module.
+use upgrade as module.
 
 
 exit=nogoals.
 order=linear.
 
 module tygron {
 	
+	%TODO: parken en groen
+	if bel(upgrades(T), not(upgrades([])))
+		then upgrade.
```

Comment:
This probably shouldn't be here, because we don't want to upgrade at this point at any moment in the game. We just need the module from this branch so we can later extend our code with the demolish functionality.